tests/ctst/world/Zenko.ts

```diff
@@ -197,37 +283,41 @@ export default class Zenko extends World {
      * @returns {undefined}
      */
     async prepareARWWI(ARWWIName: string, ARWWIPassword: string, ARWWITargetRole: string) {
+        const usedParameters = this.parameters as Parameters;
+
         if (!(ARWWIName in CacheHelper.ARWWI)) {
-            const token = await this.getWebIdentityToken(
+            const token: string = await this.getWebIdentityToken(
                 ARWWIName,
                 ARWWIPassword,
-                this.parameters.KeycloakHost || 'keycloak.zenko.local',
-                this.parameters.keycloakPort || 80,
-                `/auth/realms/${this.parameters.keycloakRealm || "zenko"}/protocol/openid-connect/token`,
-                this.parameters.keycloakClientId || Constants.K_CLIENT,
-                this.parameters.keycloakGrantType || 'password',
+                usedParameters.KeycloakHost as string || 'keycloak.zenko.local',
+                usedParameters.keycloakPort as string || '80',
+                `/auth/realms/${usedParameters.keycloakRealm as string || 'zenko'}/protocol/openid-connect/token`,
+                usedParameters.keycloakClientId as string || Constants.K_CLIENT,
+                usedParameters.keycloakGrantType as string || 'password',
             );
             this.options.webIdentityToken = token;
             if (!this.options.webIdentityToken) {
                 throw new Error('Error when trying to get a WebIdentity token.');
             }
             // Getting account ID
             const account = await SuperAdmin.getAccount({
```

Comment:
`SuperAdmin.getAccount()` returns Promise<any>, not sure what you're saying here?

Reply:
Any promise will hold a type that will be infered dynamically when you `await` the function, to avoid casting the function result to something that might not work, in case of an error, I still see that this is needed here

Reply:
`SuperAdmin.getAccount()` now returns `Promise<Command>`, as that is the most compatible return value for how the code works on cli-testing's side.